mopidy/local/search.py

```diff
@@ -81,10 +92,22 @@ def find_exact(tracks, query=None, uris=None):
                 raise LookupError('Invalid lookup field: %s' % field)
 
     # TODO: add local:search:<query>
-    return SearchResult(uri='local:search', tracks=tracks)
-
-
-def search(tracks, query=None, uris=None):
+    return SearchResult(
+        uri='local:search', tracks=tracks[offset:offset+limit])
```

Comment:
I think the missing spaces around `+` here will cause a flake8 warning with the recent flake8 update (after this PR was submitted).